en/sensor/rangefinders.md

```diff
@@ -161,6 +161,17 @@ On the Simulator you can run the commands directly in the terminal.
 
 For more information see: [Development > Debugging/Logging > Sensor/Topic Debugging using the Listener Command](../debug/sensor_uorb_topic_debugging.md).
 
+## Gazebo Simulation
+
+Lidar rangefinders can be used in the [Gazebo](../sim_gazebo_gz/index.md) simulator.
+To do this you must start the simulator using a vehicle model that includes the rangefinder.
+
+The x500 lidar model includes a 2D Lidar rangefinder:
+
+```sh
+make px4_sitl gz_x500_lidar
+```
+
```

Comment:
> 4. Also in that doc we have Gazebo Classic simulation section - https://docs.px4.io/main/en/sensor/rangefinders.html#gazebo-classic-simulation - Would it be possible for you to add a few words about simulation with Gazebo in its own section? (assuming this is the right place)?
>> i agree that it would make sense to add a gazebo section to rangefinders, but i'm not sure if i'm the right person, as i have limited expierience creating my own models.

So what I was really trying to get at here was "OK, cool, we have a simulated vehicle with a rangefinder, what can I do with it? how can I use it?

My assumption is that if I use this model PX4 will be getting rangefinder data as though there were a real rangefinder attached to the vehicle and already enabled.

On that assumption I have added this section. So basically someone else reading this doc this will assume that they can just use this lidar. Do you happen to know what range it is set up to use? Or how this can be configured?

If that is not true, then we need to know how to make it useful.

It would be good to explain how to add this same model to other simulated frames, but that is not be as important.

Reply:
yes, the only thing there is that the x500_lidar model generates a gazebo laserscan which gets translated into a obstacle distance message. not sure if this is the most representative example for rangefinder simulation... its more a collision prevention example. do we want to differentiate there?
i did add some specs on the simulated sensor. but these are hardcoded in the gazebo model. so no way to change them via parameters.